tools/internal_ci/linux/grpc_build_artifacts.sh

```diff
@@ -26,5 +26,4 @@ set -e  # rvm commands are very verbose
 rvm --default use ruby-2.4.1
 set -ex
 
-# [DO-NOT-SUBMIT]
-tools/run_tests/task_runner.py -f artifact linux presubmit -j 6
+tools/run_tests/task_runner.py -f artifact linux -j 6
```

Comment:
I just updated the parallelism to -j 12 in my recent PR, so you might wanna rebase.